RiotSharp/packages.config

```diff
@@ -1,11 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="OpenCover" version="4.6.519" targetFramework="portable45-net45+win8+wpa81" />
-  <package id="Rx-Core" version="2.2.5" targetFramework="portable-net45+win+wpa81" />
-  <package id="Rx-Interfaces" version="2.2.5" targetFramework="portable-net45+win+wpa81" />
-  <package id="Rx-Linq" version="2.2.5" targetFramework="portable-net45+win+wpa81" />
-</packages>
```

Comment:
where do all those dependencies live now? I only see newtonsoft json and reactive linq in RiotSharp.csproj?